pages/web_cloud/internet/overthebox/middle_redirection_de_port/guide.fr-fr.md

```diff
@@ -1,85 +1,132 @@
 ---
-title: 'Configurer une redirection de port sur OverTheBox'
-excerpt: "Découvrez comment rediriger un port ou une plage de ports sur votre OverTheBox"
-updated: 2023-12-21
+title: 'Comment configurer le pare-feu (firewall) sur OverTheBox ?'
+excerpt: "Découvrez comment rediriger un port ou mettre en place une DMZ sur votre OverTheBox"
+updated: 2024-11-18
 ---
 
 ## Objectif
 
-Sur l'OverTheBox, il est possible de configurer une redirection de port afin de vous permettre d'accéder à une machine de votre réseau local depuis un réseau extérieur.
+Votre équipement **OverTheBox** est protégé contre les menaces et intrusions externes, son pare-feu empêche les connexions extérieures sur votre réseau local.
+
+Ce guide vous permet de configurer le pare-feu depuis l'interface OverTheBox afin d'accéder aux équipements de votre réseau local depuis l'extérieur.
+
+Découvrez ainsi :
+
+- Comment configurer une redirection de port, pour accéder à une interface web hébergée localement.
+- Comment configurer une **DMZ**, pour utiliser votre propre routeur.
+- Comment configurer une règle pare-feu, pour par exemple restreindre la connectivité du réseau local.
+- Comment configurer un **VPN PPTP**.
 
 ## Prérequis
 
-Afin que la redirection de port fonctionne correctement, il est impératif que la machine cible de cette redirection possède comme passerelle par défaut l'IP de l' **OverTheBox**.
+- Une **OverTheBox** fournie par OVHcloud ou une installation depuis [le projet Open Source](/pages/web_cloud/internet/overthebox/advanced_installer_limage_overthebox_sur_votre_materiel).
+- Être connecté à l'interface web de l'**OverTheBox** depuis [overthebox.ovh](http://overthebox.ovh) ou [192.168.100.1](https://192.168.100.1).
+
+## En pratique
+
+### Configurer une redirection de port
 
-> [!success]
+> [!warning]
 >
-> C'est normalement déjà le cas si cette machine est configurée en récupération d'IP automatique et si le DHCP est géré par OverTheBox.
-> Pour plus d'informations, consultez la procédure d'installation : [Comment installer OverTheBox ?](/pages/web_cloud/internet/overthebox/plus_itv2_installation).
+> - Les ports **5006 à 5009** et **1090** ne peuvent pas être utilisés ou redirigés car ils sont utilisés pour le fonctionnement interne de l'OverTheBox.
+> - Il est impératif que l'appareil local cible de la redirection possède comme passerelle par défaut l'IP de l' **OverTheBox**.
 >
 
-## En pratique
+Une redirection de ports permet aux appareils externes de se connecter sur une plage de ports d'un appareil du réseau local. Dans cet exemple, nous souhaitons rendre accessible depuis un site web hébergé sur notre réseau local.
```

Comment:
@AxelDum , est-ce qu'il ne manque pas un élément dans cette deuxième phrase ? 
"Nous souhaitons rendre accessible _quelque chose_ depuis un site web hébergé sur notre réseau local."

Reply:
Indeed sorry about that, correction

Une redirection de ports permet aux appareils externes de se connecter sur une plage de ports d'un appareil du réseau local. Dans cet exemple, nous souhaitons rendre accessible depuis l'extérieur un site web      hébergé sur notre réseau local.